examples/README.md

```diff
@@ -1397,34 +1397,29 @@ Toolkit. It includes:
       machine_type: a2-highgpu-2g
   ```
 
-  Users only need to provide machine type for standard ["a2", "a3" and "g2"] machine families,
+  Users only need to provide machine type for standard ["a2", "a3" and "g2"] gpu machine families,
```

Comment:
nit as you are just updating this in this PR: 
I kind of feel like this documentation would be more visible and better placed in the node-pool readme and then linked to that from here.